openhtf/core/phase_executor.py

```diff
@@ -307,8 +307,10 @@ def _execute_phase_once(
       try:
         run_phase = phase_desc.options.run_if()
       except Exception:  # pylint: disable=broad-except
+        self.logger.debug('Phase %s stopped due to a fault in run_if function.',
+                        phase_desc.name)
```

Comment:
Please line up the start of this line with the opening parenthesis on the previous line. Make sure you're using spaces and not tabs to do the alignment.